workitems/tokenlist/l2-token-list-v1.0-psd01.md

```diff
@@ -0,0 +1,685 @@
+                            "title": "tokenId",
+                            "description": "A resolvable URI to a CanonicalTokenList entry of the token.",
+                            "type": "string",
+                            "example": ["https://polygonscan.com/address/0x0000000000000000000000000000000000001010"]
+                        },
+                        "tokenType": {
+                            "$comment": "{\"term\": \"tokenType\", \"@id\": \https://schema.org/StructuredValue\"}",
+                            "title": "tokenType",
+                            "description": "Describes the type of token.",
+                            "type": "array"
+                            "examples"[["fungible","transferable"]]
+                        },
+                        "tokenDesc": {
+                            "$comment": "{\"term\": \"tokenDesc\", \"@id\": \"https://schema.org/description\"}",
+                            "title": "tokenDesc",
+                            "description": "Brief description of the token and its functionality.",
+                            "type": "string",
+                            "examples": ["Protocol Token for the Matic Network"]
+                        },
+                        "standard": {
+                            "$comment": "{\"term\": \"standard\", \"@id\": \"https://schema.org/citation\"}",
+                            "title": "standard",
+                            "description": "Describes which standard the token adheres to.",
+                            "type": "string",
+                            "examples": ["ERC20"]
+                        },
+                        "address": {
+                            "$comment": "{\"term\": \"address\", \"@id\": \"https://schema.org/identifier\"}",
+                            "title": "address",
+                            "description": "Address of the token smart contract.",
+                            "type": "string",
+                            "examples": ["0x0000000000000000000000000000000000001010"]
+                        },
+                        "addressType": {
+                            "$comment": "{\"term\": \"address\", \"@id\": \"https://schema.org/Intangible\"}",
+                            "title": "addressType",
+                            "description": "AddressType of the token smart contract.",
+                            "type": "string",
+                            "examples": ["MaticNameSpace"]
+                        },
+                        "addressAlg": {
+                            "$comment": "{\"term\": \"addressAlg\", \"@id\": \"https://schema.org/algorithm\"}",
+                            "title": "addressAlg",
+                            "description": "Algorithm used to create the address e.g. CREATE2.",
+                            "type": "string",
+                            "examples": ["CREATE2"]
```

Comment:
is a standard address evm contract address "CREATE"?

Reply:
@realcodywburns no. but it could be called something like trimleft(keccak256(pubKey_secp256k1)),24) -> the last 40 characters/20 bytes of the Keccak-256 hash of a secp256k1 public key, or `ethStandardAddress` as a possible naming convention.

Reply:
added to description